runtime/doc/job_control.txt

```diff
@@ -102,23 +102,23 @@ function. Here's a more object-oriented version of the above:
 >
     let Shell = {}
 
-    function Shell.on_stdout(job_id, data)
+    function Shell.on_stdout(job_id, data) dict
```

Comment:
It doesn't hurt, right? I thought it better to make the examples maximally safe to copy, paste, and change for users who are just trying things out.